solr/solr-ref-guide/modules/configuration-guide/pages/coreadmin-api.adoc

```diff
@@ -318,11 +318,7 @@ http://localhost:8983/solr/admin/cores?action=RELOAD&core=techproducts
 
 [source,bash]
 ----
-curl -X POST http://localhost:8983/api/cores/techproducts -H 'Content-Type: application/json' -d '
-  {
-    "reload": {}
-  }
-'
+curl -X POST http://localhost:8983/api/cores/techproducts/reload -H 'Content-Type: application/json'
```

Comment:
[0] I'd have to check this to be certain, but we should be able to remove the `Content-type` header from this example, since this API ignores the request body.